### ISSUE TITLE
Chat UI-improvement suggestion

### ISSUE THREAD

**Issue by RolandWArnold** [coolchess123/tcec-gui#199]
Currently as new chat messages arrive, they're shown instantly, which can be hard to follow, especially when there are multiple messages arriving at around the same time.

I'd suggest using some minor animation when inserting the new divs with the new messages so it's instantly apparent as to what's changed/easy to follow the incoming messages/the conversation.

There are multiple examples online illustrating this idea, but none are perfectly along the lines of what I'm trying to describe, but even http://api.jquery.com/slideup/ illustrates the basic idea.

**Comment by JB940** [comment]
Sadly, we have no access to twitch chat and even if we hack-coded something in, iframe access is considered very bad practice.
Creating a custom chat ourselves is way too big of a task.
